src/tools/alerts.md

```diff
@@ -143,7 +143,7 @@ clicking on the plot's title. Open another tab to the Evolution View.
 
 5\. If you _still_ don't know what's going on
 
--   find a domain expert on IRC and bother them to help you out. Domain
+-   find a domain expert and bother them to help you out (see [getting help](../concepts/getting_help.md)). Domain
```

Comment:
```suggestion
-   find a domain expert and ask them to help you out (see [getting help](../concepts/getting_help.md)). Domain
```

it's not a bother, and I wouldn't want to discourage people from reaching out for help :)